libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/NavigationStyles.kt

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.navigation.ui.maps
+
+/**
+ * An object that exposes default navigation styles for day and night mode
+ */
+object NavigationStyles {
+    const val NAVIGATION_DAY_STYLE = "mapbox://styles/mapbox/navigation-day-v1"
+    const val NAVIGATION_NIGHT_STYLE = "mapbox://styles/mapbox/navigation-night-v1"
```

Comment:
```suggestion
    const val NAVIGATION_DAY_STYLE_V1 = "mapbox://styles/mapbox/navigation-day-v1"
    const val NAVIGATION_NIGHT_STYLE_V1 = "mapbox://styles/mapbox/navigation-night-v1"
```

Would we like to explicitly version the style constants? How are these exposed on iOS @mapbox/navigation-ios?

I assume we might want to obfuscate versioning and have a single constant that we can update as we go, but that could also be considered a breaking change. I know this is a grey area and was debated in the past, so I think it would be best if we follow whatever approach is already available on iOS.

Reply:
This is how iOS has it
https://github.com/mapbox/mapbox-navigation-ios/blob/992b4b5f4d63cb1a3623bef95e4bc6dd0e7d7158/Sources/MapboxNavigation/StyleURI.swift#L4-L12
as they obfuscate versioning.
cc @LukasPaczos

Reply:
Okay, let's do that as well then :+1: